### PR TITLE
feat: Preselect items left panel

### PR DESCRIPTION
**Pull request by cyaiox**
This PR preselects items in the LeftPanel when entering the ItemEditor.

https://github.com/decentraland/builder/assets/3170051/856107ea-3db4-48b6-9020-b35474978d12

### CODE REVIEW

File: src/modules/collection/utils.ts [hunk 1]
```diff
@@ -38,7 +38,10 @@ export function getSaleAddress(chainId: ChainId) {
 }
 
 export function getCollectionEditorURL(collection: Collection, items: Item[]): string {
-  return locations.itemEditor({ collectionId: collection.id, itemId: items.length > 0 ? items[0].id : undefined })
+  return locations.itemEditor({
+    collectionId: collection.id,
+    itemId: items.length > 0 ? (items.find(item => item.type === ItemType.WEARABLE) || items[0]).id : undefined
```
Comment (kevinszuchet) [inline src/modules/collection/utils.ts:43]: We already have this logic in a [location selector](https://github.com/decentraland/builder/blob/0708387ff1cf11d781dc6e38d477e9b69845332c/src/modules/location/selectors.ts#L75C6-L75C6) called `getSelectedItemId`.

WDYT about creating a new function `getFirstWearableOrItem` (or something similar), and using it in both places?

**Comment by kevinszuchet** [comment]
Would you mind reviewing why when you follow these steps:

1. Go to [collections](https://builder-5ssi9ol2a-decentraland1.vercel.app/collections).
2. Click on Open Editor.
3. Select a collection with more than 1 item.
4. The preselected item is the last instead of the first in the list.

It's unrelated to your changes, and maybe you could fix it, given that you are working on something similar.

File: src/modules/collection/utils.ts (hunk 1 above)
Comment (cyaiox) [reply src/modules/collection/utils.ts:43]: I like the idea, working on it :)

**Comment by cyaiox** [comment]
> Would you mind reviewing why when you follow these steps:
> 
> 1. Go to [collections](https://builder-5ssi9ol2a-decentraland1.vercel.app/collections).
> 2. Click on Open Editor.
> 3. Select a collection with more than 1 item.
> 4. The preselected item is the last instead of the first in the list.
> 
> It's unrelated to your changes, and maybe you could fix it, given that you are working on something similar. https://www.loom.com/share/bcacb7ac751042bfab69f430e816cef1?sid=a7748146-fd6f-4345-ade2-8b1fbee77ffc

mmm, taking a look

File: src/components/CollectionProvider/CollectionProvider.tsx [hunk 2]
```diff
@@ -65,7 +65,7 @@ export default class CollectionProvider extends React.PureComponent<Props> {
         onFetchCollection(id)
       }
       // fetch collection items if the id changes
-      this.fetchCollectionItems(itemsPage)
+      this.fetchCollectionItems(DEFAULT_ITEMS_PAGE)
```
Comment (cyaiox) [inline src/components/CollectionProvider/CollectionProvider.tsx:68]: Fetch the first items page when the collection changes

File: src/components/ItemEditorPage/LeftPanel/Collections/SidebarCollection/SidebarCollection.tsx [hunk 3]
```diff
@@ -9,12 +9,10 @@ import './SidebarCollection.css'
 
 class SidebarCollection extends React.PureComponent<Props> {
   render() {
-    const { collection, items, isSelected } = this.props
-    const collectionItems = items.filter(item => item.collectionId === collection.id).sort((a, b) => (a.createdAt > b.createdAt ? -1 : 1))
-    const itemId = collectionItems.length > 0 ? collectionItems[0].id : undefined
```
Comment (cyaiox) [inline src/components/ItemEditorPage/LeftPanel/Collections/SidebarCollection/SidebarCollection.tsx:14]: Getting the id using the `getSelectedItemId from modules.location.selectors.ts`

**Review by kevinszuchet — APPROVED** [review]
LGTM 😍

**Pull request merged**